src/masternodeman.cpp

```diff
@@ -205,7 +205,10 @@ void DumpMasternodes()
 CMasternodeMan::CMasternodeMan():
         cvLastBlockHashes(CACHED_BLOCK_HASHES),
         nDsqCount(0)
-{}
+{
+    for (int i = 0; (unsigned) i < CACHED_BLOCK_HASHES; i++)
+        cvLastBlockHashes.Set(i, UINT256_ZERO);
```

Comment:
Can make it cleaner adding another constructor to the `CiclycVector` class:
```
CyclingVector(unsigned int _MAX_SIZE, T defaultValue):
        MAX_SIZE(_MAX_SIZE), vec(MAX_SIZE, defaultValue) { }
```

Reply:
Done.